planner/optimizer/src/main/java/eu/seaclouds/platform/planner/optimizer/Optimizer.java

```diff
@@ -74,8 +71,7 @@ public Optimizer(int num, SearchMethodName name) {
 
          try {
             log.error("Calling a Replanning. The previously generated Plan will be used as a base");
-            outputPlans = optimizerReplanning.optimize(appModel,
-                  suitableCloudOffer, NUMBER_OF_PLANS_GENERATED);
+            outputPlans = optimizerReplanning.optimize(appModel, suitableCloudOffer, NUMBER_OF_PLANS_GENERATED);
             previousPlans = outputPlans;
          } catch (Error E) {
             log.error("Error optimizing the Replanning");
```

Comment:
Right. Done.
I've modified it (not yet pushed). Now the the same error is thrown in the replanning case (infact it is not implemented yet the replannig). 

In case of the initial optimizer (line 66), I left the error still masked. I think this is still OK, as the this optimize method is the main one called (as a service in the near future), masking the error but returning empty plans I think is a good decision at this moment. 

Let me know if you disagree on this.